bip-tap-universe.mediawiki

```diff
@@ -242,16 +267,26 @@ happened.  Importantly, one cannot prove that a Multiverse has complete
 history, as a Multiverse can only commit to what it directly observed, or was
 shown to it.
 
+Similar normal Universes, a Multiverse can either commit to the set of Issuance
+Universes for a set of assets, or the set of Transfer Universes for the set of
+assets. Once can be used to bundle the bootstrap of a bundle of assets, while
+the other can be used to store the complete transfer history of a set of
+assets.
+
 A Multiverse has the following structure:
 * Similar to normal Taproot Asset commitments, the Multiverse itself contains two nested MS-SMT trees. The upper tree commits to the set of asset groups observed, with the inner tree committing to the transaction history of each of the asset groups.
 * Upper tree structure:
 ** <code>key</code>: <code>asset_id</code> or <code>sha256(asset_key_family)</code>
 ** <code>value</code>: <code>asset_group_tree_root</code>
-** <code>sum_value</code>: <code>asset_group_sum</code>
+** <code>sum_value</code>::
+*** For ''Transfer Universes'': a value of `1`
+*** For ''Issuance Universes'': the <code>sum_value</code> of the Transfer Universe root, this sums to the total number of transactions in the Multiverse.
```

Comment:
So the top level sum would be the sum of all transactions in the Multiverse plus the number of assets minted? Since we use the value of `1` for transfer universes? So we'd add the number of universes that registered at least one transfer to the number of transactions?
Or I'm probably misunderstanding how these sums actually work. Just wanted to make sure this part and the part below are what we've currently implemented.

Reply:
Sum for issuance multiverse: the total number of assets tracked
Sum for transfer multiverse: the total number of transfers across all assets tracked

Reply:
Yeah we'll have one eventually for Pocket universes, and also the commitment for canonical universes on chain. For that commitment tho, we can just have the two hashes, then an omitted value reserved for later to have room to tweak things in the future.